python/taichi/snode/fields_builder.py

```diff
@@ -71,49 +71,60 @@ def empty(self):
     def finalized(self):
         return self._finalized
 
+    def deactivate_all(self):
+        """Same as :func:`taichi.lang.snode.SNode.deactivate_all`"""
+        if self._finalized:
+            self._root.deactivate_all()
+        else:
+            warning(
+                """'deactivate_all()' would do nothing if FieldsBuilder is not finalized.
+    Due to 'ti.root' is implemented by 'ti.FieldsBuilder' implicitly, if you are
+    calling 'ti.root.deactivate_all()', please using 'ti.deactivate_all_snodes()'
+    instead.""")
+
     def dense(self, indices: Union[Sequence[_Axis], _Axis],
               dimensions: Union[Sequence[int], int]):
-        """Same as :func:`taichi.lang.SNode.dense`"""
+        """Same as :func:`taichi.lang.snode.SNode.dense`"""
```

Comment:
have you tested the docstring link here?

Reply:
It works